### PR TITLE
Select all

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

File: app/scripts/proactive/view/dom.js [hunk 1]
```diff
@@ -1016,17 +1016,16 @@ define(
 
 
             var ctrlDown = false;
-            var ctrlKey = 17, commandKey = 91, vKey = 86, cKey = 67, zKey = 90, yKey = 89;
+            var ctrlKey = 17, commandKey = 91, vKey = 86, cKey = 67, zKey = 90, yKey = 89, aKey = 65;
             var pasteAllow = true;
             var canDoPast = false
-
-            $(document).keydown(function (e) {
+            $('#workflow-designer-outer').bind('keydown', function (e) {
```
Comment (fviale) [inline app/scripts/proactive/view/dom.js]: I am afraid by doing this change you disable copy/paste at all outside of the canvas

Comment (medou-boushab) [reply app/scripts/proactive/view/dom.js]: No, it still works

Comment (fviale) [reply app/scripts/proactive/view/dom.js]: you are right, just testing it right now and it seems to work as expected. Before approve it, I will check with you tomorrow so I can understand the changes.

File: app/scripts/proactive/view/dom.js [hunk 2]
```diff
@@ -1019,14 +1019,13 @@ define(
...
                 if (e.keyCode == ctrlKey || e.keyCode == commandKey) ctrlDown = true;
             }).keyup(function (e) {
                 if (e.keyCode == ctrlKey || e.keyCode == commandKey) ctrlDown = false;
             });
 
-            $(document).keydown(function (e) {
+            $('#workflow-designer-outer').bind('keydown',function (e) {
```
Comment (fviale) [inline app/scripts/proactive/view/dom.js]: could you change the title of the pop-up ?
instead of "alert-success" => "Copy / Paste"

Comment (medou-boushab) [reply app/scripts/proactive/view/dom.js]: done

**Pull request merged**